src/coord/src/sink_connector.rs

```diff
@@ -339,7 +339,7 @@ async fn build_kafka(
     }
     let client = config
         .create::<AdminClient<_>>()
-        .expect("creating admin client failed");
+        .context("creating admin client failed")?;
```

Comment:
I'm sorry I haven't looked super closely but where would we see the output of these context calls? It doesn't seem like it makes its way into the error text returned by the client.

Reply:
It does, in the psql client it looks like this:

```
ERROR:  creating admin client failed: Client config error: Invalid value "FOO" for configuration property "security.protocol" security.protocol FOO
```